atomic_reactor/utils/operator.py

```diff
@@ -200,6 +199,19 @@ def check_csv(data, schema):
     validate_with_schema(data, schema)
 
 
+def modify_dict_recursively(target, mods, append=False):
+    for key, value in mods.items():
+        if isinstance(value, dict):
+            # recursive call
+            target[key] = modify_dict_recursively(target.get(key, {}), value, append=append)
+        else:
+            if append:
+                target.setdefault(key, []).extend(value)
+            else:  # Got `None`, create key & value
+                target[key] = value
```

Comment:
It's just a minor suggestion, but why not use elif statement here?

Reply:
`elif` where?

Reply:
```suggestion
        elif append:
            target.setdefault(key, []).extend(value)
        else:  # Got `None`, create key & value
            target[key] = value
```

Reply:
Inaccurate comment